### DISCUSSION TITLE
Theme Color doesn't highlight in the CSS file.

### DISCUSSION THREAD

**Discussion by AbdullahKhan780** [shadcn-ui/ui#937] [General]
You should wrap hsl() color function inside the global CSS file instead of wrapping it in tailwind.config.ts. Because VS code doesn't highlight the color. It will help the users to easily modify the color theme.

**Comment by lloydrichards** [comment]
>CSS variables must be defined without color space function. See the [Tailwind CSS documentation](https://tailwindcss.com/docs/customizing-colors#using-css-variables) for more information.

There is a note in the [Theme page](https://ui.shadcn.com/docs/theming#:~:text=CSS%20variables%20must%20be%20defined%20without%20color%20space%20function.%20See%20the%20Tailwind%20CSS%20documentation%20for%20more%20information.) about why you shouldn't use CSS color namespaces since it will prevent tailwind from adding alpha values.  This kind of depends on your project, but several of the hover and focus effects in shadcn/ui are using something like `bg-primary/80` which won't work with the hsl() in the css

**Reply by AbdullahKhan780** [reply]
Thank you @lloydrichards, for the prompt response!
